docs/command_line_tool/phylum_project_set-thresholds.md

```diff
@@ -3,17 +3,34 @@ title: phylum project set-thresholds
 category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
+
 Interactively set risk domain thresholds for a project
+
 ```sh
-phylum project set-thresholds [OPTIONS] <name>
+Usage: phylum project set-thresholds [OPTIONS] <name>
 ```
-Resulting scores at or below the defined threshold will fail.
```

Comment:
Another lost detail.

Reply:
Expanded it a bit and moved it to a separate details section. Hopefully should be a bit more clear and not collide with the new template system.